SyncKusto/MainForm.cs

```diff
@@ -365,11 +365,20 @@ void WriteToTarget(IKustoSchema schema)
                 }
             }
 
+            DialogResult dialogResult = DialogResult.None;
+
             void DeleteFromTarget(IKustoSchema schema)
             {
                 if (spcTarget.SourceSelection == SourceSelection.Kusto())
                 {
-                    schema.DeleteFromKusto(kustoQueryEngine);
+                    if (dialogResult == DialogResult.None)
```

Comment:
I like the idea of adding a warning in, but I'm concerned about this being slow for experienced users. I'll chat with the team on Monday and think about this idea and see what we can do to get this plugged in. Thanks for the contribution!

Reply:
Cool, thanks!

Reply:
Ok my github skills are apparently lacking. I ended up creating a new pull request (https://github.com/microsoft/synckusto/pull/8) but I couldn't figure out how to add you as a reviewer. Please feel free to leave comments and this PR can be abandoned. Thanks for the feedback!